src/status_im/chat/handlers/receive_message.cljs

```diff
@@ -22,9 +22,16 @@
 
 (declare add-message-to-wallet)
 
+;; XXX: Delete me once clock semantics work
```

Comment:
Yes, just including it for now while it is a WIP; will remove this before printing.

Will add more comments in the code itself!